### PR TITLE
Add timer frontend

### LINKED ISSUE #8: Add timer frontend
Depends on:
- [x] #7

---

It would kinda help if there was an actual timer displayed.

It doesn't need to look pretty for now, after the subathon it can always be changed to look decent.
Controls would also help (starting, pausing, stopped)

### LINKED ISSUE #9: Add timer adjustments for Twitch events
The timer kinda needs to increase whenever an event happens on Twitch (sub, bits, etc.)

This and #1 have been split into two issues to let #7 get progress done.

---

### Pre-requisites:
- [x] #10

### PR DESCRIPTION
**Pull request by gtaEPIC**
Currently, if there's a timer, it and it is told to count, it will count down. I couldn't test with the server, but theoretically, the socket.io stuff should work too.
- Closes #8
- Closes #9

### CODE REVIEW

**Comment by gtaEPIC** [comment]
Oh 1 issue I realized, not sure how it'll be solved. Currently the only way the socket.io library is being loaded is with the actual js library. Because I couldn't get it to pull from the non-existing server, I had to download the library for it. (And I can't send a commit with it cause of the .gitignore)

**Comment by gtaEPIC** [comment]
Cool, that should work, not sure if it will. Feel free to test it

**Review by sh0ckR6 — CHANGES_REQUESTED** [review]
Yeah so connecting to the WebSocket just doesn't work. I had to mess around with the express server/filesystem layout to get this to work. I should have a fix committed pretty quick for this. Other than that, looks fine!

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        margin: 4px 2px;
+        cursor: pointer;
+        -webkit-transition-duration: 0.4s; /* Safari */
+        transition-duration: 0.4s;
+    }
+    </style>
+    <script src="/socket.io/socket.io.js"></script>
+    <script>
+
+        const socket = io();
```
Comment (sh0ckR6) [inline frontend/index.html:68]: There's actually no connection being made, it doesn't know what server URL it should be looking for here.

**Pull request merged**